.github/workflows/publish.yml

```
@@ -22,50 +21,80 @@ jobs:
+            - name: Install poetry
+              run: pipx install poetry
+
             - name: Use Node.js
               uses: actions/setup-node@v4
               with:
                 node-version: "22.x"
                 cache: npm
 
             - name: Update version before publishing
-              run: poetry version ${{ steps.extract_version.outputs.version }}
+              run: poetry version ${{ steps.extract_version.outputs.version || github.event.inputs.version }}
```

actionlint (1.7.8)

property "extract_version" is not defined in object type {bump_rc: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)